portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -54,8 +53,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           Comment comment = new Comment(name, content, timestamp);
           comments.add(comment);
 
-          displayNum--;
-          if (displayNum == 0)
+          if (numComments <= comments.size())
             break;
```

Comment:
In java do if statements need curly braces around what they should do?

Reply:
if it's only one line they don't need curlies